src/WebJobs.Script/Diagnostics/HostErrorLogger.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics
+{
+    public class HostErrorLogger : ILogger
```

Comment:
Not really sure why we need a separate logger for errors - why can't the regular host logger handle dedupe of errors itself? Granted, I may not have a complete mental model of your logger pipeline and how this fits in.

Reply:
This is a 1:1 copy from TraceWriter. I'm honestly not sure what this thing is for -- none of our tests hit anything significant in here that I could find, but I didn't want to remove it in case something broke. https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L1677

Is that for Javascript, maybe? We don't have all of our end-to-end tests running in v2 yet so it's possible that it just isn't being exercised.

I'll give it a try in v1 and see what scenarios hit this code.